script/FullDeployer.s.sol

```diff
@@ -88,7 +90,8 @@ contract FullDeployer is ExtendedHubDeployer, ExtendedSpokeDeployer, AdaptersDep
             centrifugeId: centrifugeId,
             adminSafe: ISafe(vm.envAddress("ADMIN")),
             maxBatchGasLimit: uint128(maxBatchGasLimit),
-            version: keccak256(abi.encodePacked(vm.envOr("VERSION", string(""))))
+            // Default to bytes32(0) version to be consistent with CommonDeployer
+            version: bytes(versionString).length > 0 ? keccak256(abi.encodePacked(versionString)) : bytes32(0)
```

Comment:
I think this is not need it? If `versionString` is length == 0, then it's the same as `bytes32(0)`, right?

Reply:
Correct! Thanks for the explanation